src/plot/ROOTMultiPlot.cpp

```diff
@@ -77,7 +77,10 @@ ROOTMultiPlot::Construct(){
   for(size_t i = 0; i < fNames.size(); i++){
     fLegend.AddEntry(&fHists.at(i), fNames.at(i).c_str());
     fHists[i].SetLineColor(fColorScheme[i %fNcolors]);
-    fHists.at(i).Draw("same");
+    if(i!=0)
```

Comment:
Ah ha I see. Well looking at it now i think fConstructed is a flag that stops that happening, and Construct() itself is private.. 

But at the moment, if you want to add an extra pdf after drawing it wont do that. I guess it needs a clear() method that wipes the canvas and sets fConstructed to false.